### PR TITLE
expose trackVerified()

### PR DESCRIPTION
**Pull request by nickpatrick**
Exposes `trackVerified()`, which calls `POST /track` on desktop agent running on `localhost:52516`. Need to add error handling (e.g., case where desktop agent not installed or not running)

### CODE REVIEW

**Comment by tderouin** [comment]
I tested this out with Radar Verify, and it worked well.

File: src/index.ts [hunk 1]
```diff
@@ -143,6 +144,14 @@ class Radar {
     }
   }
 
+  public static trackVerified(params: RadarTrackParams = {}) {
+    try {
+      return VerifyAPI.trackVerified(params);
+    } finally {
+      ConfigAPI.getConfig(params); // call with updated permissions
```
Comment (kochis) [inline src/index.ts:151]: It doesn't seem like trackVerified is making any calls that would modify permissions

File: src/http.ts [hunk 2]
```diff
@@ -27,10 +27,12 @@ class Http {
     method,
     path,
     data,
+    host = undefined,
   }: {
     method: HttpMethod;
     path: string;
     data?: any;
+    host?: string | undefined;
```
Comment (kochis) [inline src/http.ts:35]: I think this can just be `host?: string;`, then you don't need the default argument above.

File: src/index.ts (hunk 1 above)
Comment (nickpatrick) [reply src/index.ts]: Agreed, will simplify

File: src/http.ts [hunk 3]
```diff
@@ -129,7 +132,11 @@ class Http {
       }
 
       xhr.onerror = function() {
-        reject(new RadarServerError());
+        if (host && host.includes('localhost')) {
```
Comment (kochis) [inline src/http.ts:135]: We should include the port here, it's possible that localhost is valid (ie. developing against dashboard locally)

File: src/index.ts (hunk 1 above)
Comment (nickpatrick) [reply src/index.ts]: Done

File: src/http.ts (hunk 2 above)
Comment (nickpatrick) [reply src/http.ts]: Done

**Review by kochis — APPROVED** [review]

**Pull request merged**